### DISCUSSION TITLE
建议添加github贡献日历模块

### DISCUSSION THREAD

**Discussion by xianyuxianjun** [halo-dev/halo#6362] [Ideas]
### Your current Halo version

2.18

### Describe this feature

我想大部分使用halo建站的人都是程序员而且应该相当一部分是来做博客的，那么给博客添加一个github贡献日历模块应该是很合理的。

**Comment by guqing** [comment]
你可以写一个插件来做这件事，目前 Halo 没有计划添加这样的功能
